packages/fuel-indexer-tests/assets/fuel_indexer_test.yaml

```diff
@@ -2,7 +2,7 @@ namespace: fuel_indexer_test
 graphql_schema: packages/fuel-indexer-tests/assets/fuel_indexer_test.graphql
 abi: packages/fuel-indexer-tests/contracts/fuel-indexer-test/out/debug/fuel-indexer-test-abi.json
 start_block: 1
-contract_id: fuel1mm5dud7hsw2u0nj0n60yr20ag4gewezpg7au9twgdgpupcup6ajsge77vg
+contract_id: ~
```

Comment:
@deekerno Let's either add this back or pull in changes from https://github.com/FuelLabs/fuel-indexer/pull/570 and take the incoming change from the merge conflict?

Other than that PR is fine and will ✅ when CI passes